conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -137,6 +157,59 @@ class GramineEnclaveHandle(
         }
     }
 
+    private fun prepareCommandToRun(port: Int): List<String> {
+        val gramineCommand = getGramineExecutable(enclaveMode)
+        val javaCommand = getJavaCommand(port)
+
+        val command = if (isPythonEnclave()) {
+            listOf(gramineCommand) + javaCommand
+        } else {
+            val user = runSimpleCommand("id", "-u")
+            val group = runSimpleCommand("id", "-g")
+            val dockerCommand = getDockerCommand(user, group)
+            dockerCommand + listOf(gramineCommand) + javaCommand
+        }
+        logger.debug("Running enclave with command: ${command.joinToString(" ")}")
+        return command
+    }
+
+    private fun getJavaCommand(port: Int): List<String> {
+        return listOf(
+            "java",
+            "-cp",
+            GRAMINE_ENCLAVE_JAR,
+            "com.r3.conclave.enclave.internal.GramineEntryPoint",
+            port.toString()
+        )
+    }
+
+    private fun getDockerCommand(user: String, group: String): List<String> {
+        javaClass.copyResource("/$GRAMINE_SECCOMP", workingDirectory.toAbsolutePath() / GRAMINE_SECCOMP)
+        val workingDirectoryPath = workingDirectory.toFile().absolutePath
+        return listOf(
+            "docker",
+            "run",
+            "-u",
+            "$user:$group",
+            "--network",
+            "host",
+            "--device=/dev/sgx_enclave",
+            "--device=/dev/sgx_provision",
+            "-v",
+            "/var/run/aesmd:/var/run/aesmd",
```

Comment:
Can you put these two together on the same line to make it clearer they are needed together

Reply:
Good idea, will do the same for `-u` where I used it